gulpfile.babel.js

```diff
@@ -50,9 +57,120 @@ gulp.task("server", ["hugo", "css", "js"], () => {
   gulp.watch("./site/**/*", ["hugo"]);
 });
 
+gulp.task("new-incident", (cb) => {
+  const file = fs.readFileSync("site/config.toml").toString();
+  const config = toml(file);
+
+  const questions = [{
+    type: "input",
+    name: "name",
+    message: "What is the cause of the incident?",
+    filter: ((name) => kebabCase(name)),
```

Comment:
hmm, i didn't realize that... If you use punctuation in the name it uses that in the file name also. I think that would break on windows for some use cases. 

Since hugo uses the file name for the title they are stuck together. I bet we could split those if we changed the archetype template. That might be the best solution. I'll look into it.

Reply:
The template has the kebab replacing going on inside of it already. I should just remove it.